Neos.Flow/Classes/Configuration/ConfigurationSchemaValidator.php

```diff
@@ -92,7 +92,10 @@ protected function validateSingleType(string $configurationType, string $path =
     {
         $availableConfigurationTypes = $this->configurationManager->getAvailableConfigurationTypes();
         if (in_array($configurationType, $availableConfigurationTypes) === false) {
-            throw new Exception\SchemaValidationException('The configuration type "' . $configurationType . '" was not found. Only the following configuration types are supported: "' . implode('", "', $availableConfigurationTypes) . '"', 1364984886);
+            throw new Exception\SchemaValidationException(
+                $this->translator->translateById('configuration.anErrorOccurredDuringValidationOfTheConfiguration.body', [$configurationType,implode('", "', $availableConfigurationTypes)], null, null, 'Main', 'Neos.Flow'),
```

Comment:
Where does the `$this->translator` instance come from? I can't see it initialized in the object

Reply:
good point, my bad. Commited a Bugfix to add the missing translator.